### ISSUE TITLE
Fine tuning recognizer for SROIE 2019 dataset -- receipts

### ISSUE THREAD

**Issue by hadils** [faustomorales/keras-ocr#136]
I am using ```keras_ocr``` for doing receipt scanning and using the SROIE 2019 receipt dataset. I am finding that the recognizer underfits the data -- the loss and val_loss plateau around 11.0. I would greatly appreciate any help given since I don't have anyone to discuss the problem with...

Thanks,
Hadil
